src/visualizer/sortingvisualizer/Visualizer.js

```diff
@@ -156,12 +167,20 @@ const Visualizer = () => {
     setAnimationPercentage(animationPx);
   };
 
+  /**
+   * Resets the states of the "blocks" or "oval" when the sorting animation is done
+   *
+   * @param finalReferenceArray The end state of the array holding the states of each block.
+   */
   const resetDataWhenAnimationFinish = (finalReferenceArray) => {
     setIsPlay(false);
     setIsReplay(true);
     setReferenceArray(resetArray(visualizerAlgorithm, finalReferenceArray));
   };
 
+  /**
+   * A object contains values to be passed around the other components via React's context
```

Comment:
I think for variables we should include the type? I use VS code for jsdoc, and it seems to provide suggestions to include type of variable. 

![image](https://user-images.githubusercontent.com/65328119/103145151-3cfc2980-4770-11eb-840f-9088066667b7.png)

You can view here for more info: https://jsdoc.app/tags-type.html